BusinessLayer/Services/WordService.cs

```diff
@@ -107,5 +104,14 @@ public IEnumerable<WordDTO> GetByTagName(string tag)
         {
             return _mapper.MapToDTO( _repoWrapper.Word.FindByCondition(w => w.WordTags.Any(wt => wt.Tag.Name.Equals(tag))));
         }
+
+        public long GetCountByName(string name, int skipNumber)
+        {
+            return _repoWrapper.Word
+                .FindByCondition(w => w.Name.Contains(name) && w.WordStatus.Equals(WordStatus.Сonfirmed))
+                .OrderByDescending(w => w.LikesCount)
+                .ThenBy(w => w.DislikesCount)
+                .Skip(skipNumber).Count();
+        }
```

Comment:
It is a code duplication, all that I can is make method to return a part of query.